packages/celotool/package.json

```diff
@@ -38,8 +38,7 @@
     "@types/web3": "^1.0.18",
     "@types/yargs": "^12.0.1",
     "jest": "^24.8.0",
-    "ts-jest": "^24.0.0",
-    "ts-node": "^7.0.1"
```

Comment:
why remove ts-node?

Reply:
I moved everything to the root package.

For devDependencies that makes sense. It's easier to manage, and to have the same version across every project